shcheck/shcheck.py

```diff
@@ -308,6 +320,11 @@ def main():
         json_results["present"] = {}
         json_results["missing"] = []
 
+        # Before parsing, remove X-Frame-Options if there's CSP with frame-ancestors directive
+        if "frame-ancestors" in headers.get('Content-Security-Policy'.lower()).lower():
+            sec_headers.pop("X-Frame-Options")
+            headers.pop("X-Frame-Options".lower())
```

Comment:
@santoru : I have the problem that I have CSP with frame-ancestors directive configured but no `X-Frame-Options` header. That should be a valid scenario, I guess. But this raises A KeyError: 
<img width="481" alt="image" src="https://github.com/santoru/shcheck/assets/11364625/96309f4f-21a5-4b05-994a-820e863b7db3">

Reply:
Apologies, should be fixed in new version (v1.6.4). Can you please confirm?

Reply:
@santoru : perfect, works! Thanks for this super fast fix!

Reply:
Thank you!